tsl/test/isolation/specs/compression_ddl.spec

```diff
@@ -29,7 +29,7 @@ step "Sc" {COMMIT;}
 
 session "LC"
 step "LockChunk1"   { BEGIN; select lock_chunktable (q.chname) from (select ch.schema_name || '.' || ch.table_name as chname from _timescaledb_catalog.hypertable ht, _timescaledb_catalog.chunk ch where ch.hypertable_id = ht.id and ht.table_name like 'ts_device_table' order by ch.id limit 1 ) q; }
-step "UnlockChunk" {ROLLBACK;}
+step "UnlockChunk" { ROLLBACK;}
```

Comment:
looks like this change is not required.

Reply:
Since I will need to rebase, I will remove this change.